Core/include/Acts/Geometry/GeometryIdentifier.hpp

```diff
@@ -33,9 +32,11 @@ class GeometryIdentifier {
   using Value = uint64_t;
 
   /// Construct from an already encoded value.
-  constexpr GeometryIdentifier(Value encoded) : m_value(encoded) {}
+  constexpr GeometryIdentifier(Value encoded)
+      : m_value(encoded), m_misalignmentX(0.0), m_misalignmentY(0.0) {}
```

Comment:
`GeometryIdentifier` will need to contain it's internal structure as a 64bit integer, so we'll have to find another place to put this information.